@navikt/core/react/src/util/i18n/locales/en.ts

```diff
@@ -0,0 +1,27 @@
+import type nb from "./nb";
+
+export default {
+  FileUpload: {
+    dropzone: {
+      button: "Select file",
```

Comment:
Would "Pick" be more correct here?
```suggestion
      button: "Pick file",
```

Reply:
Same for "buttonMultiple" if so

Reply:
(just some screengrabs from the MDN input type=file)
Edge / Chrome:
<img width="240" alt="image" src="https://github.com/user-attachments/assets/1bb47190-bbcc-4e13-9fa3-27745072434c">
FF:
![image](https://github.com/user-attachments/assets/04c876ed-bc74-47b8-b2b6-65327ea1a14c)

I think `Choose` works well (given there's no consensus out there, I'll just _choose_ the one I like the best 😂 )

Reply:
For the casing, I don't think it makes a huge difference, for a button It might seem appropriate to use title casing (when each word is capitalized except for the "plumbing" words like `for`, `of`, `and` etc). Since it then feels a bit more emphasized and distinct from prose tekst. Since buttons are usually supposed to be short and emphasized, but I'm also not really sure about conventions for casing on more specific features of the web 😅. It feels like Title case makes sense though.